pkg/github/github.go

```diff
@@ -93,13 +108,110 @@ func NewClient(ctx context.Context, token string, opts ...Option) *GitHubClient
 	return g
 }
 
+type Issue struct {
```

Comment:
Can you move these to the top, it feels strange placed in the middle of the file like this (both Issue and IssueComment)

Reply:
Done